GameData/RealismOverhaul/RO_SuggestedMods/KODS/RO_Electron.cfg

```diff
@@ -0,0 +1,510 @@
+				PROPELLANT
+				{
+					name = MMH
+					ratio = 0.5
+				}
+				PROPELLANT
+				{
+					name = NTO
+					ratio = 0.5
+				}
+				PROPELLANT
+				{
+					name = Helium
+					ratio = 11.25
+					ignoreForIsp = True
+				}
+				atmosphereCurve
+				{
+					key = 0 280
+					key = 1 100
+				}
+			}
+		}
+	}
+	@MODULE[ModuleRCSFX]
+	{
+		!resourceName {}
+		!atmosphereCurve {}
+		//	Unspecified Green Propellant
+		PROPELLANT
+		{
+			ratio = 1.0
+			name = ASCENT
+			DrawGauge = True
+		}
+		PROPELLANT
+		{
+			name = Helium
+			ratio = 11.25
+			ignoreForIsp = True
+		}
+		atmosphereCurve
+		{
+			key = 0 254.6
+			key = 1 91.5
+		}
+	}
+	@MODULE[ModuleDecouple]
+	{
+		@ejectionForce = 20
+	}
+}
+
+
+//	Photon Explorer
+@PART[PhotonExplorer]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.28
+	@title = Photon Explorer
+	@manufacturer = #roMfrRocketLab
+	@description = A larger satellite bus made by Rocket Lab. Can fly both on Electron and on other launch vehicles. Variants of this bus have powered NASA's CAPSTONE Spacecraft to Cislunar Space and are the basis for the ESCAPADE Mars Orbiters.
+	@mass = 0.047
+	%skinTempTag = HRSI
+	%internalTempTag = Molybdenum //	idk just a placeholder for composits
+	!RESOURCE[*],* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = SM-IV
+		volume = 255 //	210kg
+		basemass = -1
+	}
+	RESOURCE
+	{
+		name = ElectricCharge
+		amount = 1500
+		maxAmount = 1500
+	}
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
+		configuration = Hyper Curie
+		CONFIG
+		{
+			name = Hyper Curie
+			specLevel = operational
+			minThrust = 0.12
+			maxThrust = 0.12
+			heatProduction = 100
+			PROPELLANT
+			{
+				name = MMH
+				ratio = 0.5
+			}
+			PROPELLANT
+			{
+				name = NTO
+				ratio = 0.5
+			}
+			PROPELLANT
+			{
+				name = Helium
+				ratio = 11.25
+				ignoreForIsp = True
+			}
+			atmosphereCurve
+			{
+				key = 0 310
+				key = 1 100
+			}
+		}
+	}
+	@MODULE[ModuleRCSFX]
+	{
+		!resourceName {}
+		!atmosphereCurve {}
+		//	Unspecified Green Propellant, MMH+NTO Placeholder
+		PROPELLANT
+		{
+			name = MMH
+			ratio = 0.5
+		}
+		PROPELLANT
+		{
+			name = NTO
+			ratio = 0.5
+		}
+		PROPELLANT
+		{
+			name = Helium
+			ratio = 11.25
+			ignoreForIsp = True
+		}
+		atmosphereCurve
+		{
+			key = 0 310
+			key = 1 100
+		}
+	}
+	@MODULE[ModuleDecouple]
+	{
+		@ejectionForce = 20
+	}
+}
+
+
+//	Rutherford SL
+@PART[Rutherford]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.28
+	%engineType = RutherfordEngine
+}
+
+
+//	Rutherford Cluster
+@PART[RutherfordCluster]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.28
+	%engineType = RutherfordEngine
+	%engineTypeMult = 9
+	%clusterMultiplier = 1.5
+}
+@PART[RutherfordCluster]:AFTER[RealismOverhaulEngines]
+{
+	@title = Rutherford Engine Cluster
+	@MODULE[ModuleEngineConfigs]
+	{
+		@origMass = 0.1 // Add Engine Shielding, QD attachments, plumbing, etc
```

Comment:
There are more elegant ways to do this, see https://github.com/KSP-RO/RealismOverhaul/blob/b66997d58c1bb992f3d5ebc16fce0c32bca21326/GameData/RealismOverhaul/Engine_Configs/EngineVariants_Config.cfg#L76